scripts/morse.js

```diff
@@ -0,0 +1,86 @@
+
+/*----------  Imports  ----------*/
+import * as c from './constants.js'
+import * as gu from './generalUtils.js'
+
+/*----------  Function Declaration  ----------*/
+
+export const translateEToMC = (inputString) => {
```

Comment:
this is a bit prone to making a typo that is hard to pick up. 
```englishToMorse``` will be easier to notice if there's a wrong capitalisation somewhere